requirements/base.in

```diff
@@ -1,10 +1,10 @@
 # Core requirements for using this application
 
 celery                                  # Run task workers in other locations
-Django>=1.8,<1.11                       # Web application framework
+Django>=1.8,<2.0                        # Web application framework
```

Comment:
Hmm, it may, but I'm worried it may have the negative effect of confusing someone who's well aware of the next planned version. If @brittneyexline agrees to make it `1.12`, I'll go ahead with that, not a big deal.